### DISCUSSION TITLE
Is this supposed to include all sorts of text/voice to speech related funtions like traning, 'inference', etc.

### DISCUSSION THREAD

**Discussion by dadadies** [rsxdalv/TTS-WebUI#316] [General]
Im new to AI voice, tts, voice-to-voice, etc. and been looking at RVC and tortoise and this seems like this includes those and more. Is this like a UI for all of them?

**Comment by rsxdalv** [comment]
There are very many projects, it's a bit different from stable-diffusion where there's _one_, instead there are a dime a dozen. I have included some that are noteworthy and compatible within a single project.
However, giving access to many projects means I haven't implemented all the details, such as training for RVC or Tortoise. But if you want to do training, you can take the finished models and use them here as well.
